crates/teloxide-core/src/types/message.rs

```diff
@@ -1430,13 +1430,13 @@ mod tests {
                 "type": "regular",
                 "thumb": {
                     "file_id": "AAMCAgADGQEAARIt0GMwiZ6n4nRbxdpM3pL8vPX6PVAhAAIjAAOw0PgMaabKAcaXKCABAAdtAAMpBA",
-                    "file_unique_id": "AQADIwADsND4DHI",
+                    "file_unique_id": "AQADIwADsAND4DHI",
```

Comment:
These IDs trigger false positives... What if I actually apply the suggestions that `typos` CLI makes for them instead of ignoring some files? These IDs are used in tests only, so I suppose there won't be anything bad if we edit them to make `typos` happy?

Reply:
I guess? Although this is weird